mail_reply/README.rst

```diff
@@ -0,0 +1,16 @@
+===================
+ Show reply button
+===================
+
+The module adds the reply button on the message is not attached to the recording.
```

Comment:
The module allows to reply to any message. Out-of-box odoo shows reply button for messages attached to some record.